Cargo.toml

```diff
@@ -24,3 +24,6 @@ members = [
     "pgx-examples/strings",
     "pgx-examples/triggers",
 ]
+
+[profile.dev.build-override]
```

Comment:
It seems this is dead code or accidentally left in?

Reply:
Sorry, I had a pending review of my own that commented on this.  See this comment:  https://github.com/tcdi/pgx/pull/526#discussion_r841260088